table/table.go

```diff
@@ -7,6 +7,10 @@ import (
 	"github.com/charmbracelet/x/ansi"
 )
 
+// HeaderRow denotes the header's row index used when rendering headers. Use
+// this value when looking to customize header styles in StyleFunc.
+const HeaderRow int = -1
```

Comment:
Does this need to be exported?

Reply:
Yeah this is the value they would be using in `StyleFunc` to style the header instead of 0. I figured better to export a const than have them check for -1